cypress/README.md

```diff
@@ -7,22 +7,41 @@
 4. Run storybook `npm run storybook`
 5. Open new terminal at the same path
 6. Run cypress `npm run test-cypress`
-7. If you would like to run cypress in command line (headless browser for continous integration) use `npm run test:ci`
-7.1 If you would like to run specific cypress tests in command line (headless browser for continous integration) use:
-  - for Windows `npm run test:ci --spec 'cypress/features/[tests-type]/[featureFileName].feature'`
-  - for MacOS `npm run test:ci -- --spec 'cypress/features/[tests-type]/[featureFileName].feature'`
+7. If you would like to run cypress in command line (headless browser for continous integration) use `npm run test-cypress`
```

Comment:
remove this line because `npm run test-cypress` will open full cypress window UI

Reply:
fixed